examples/src/main/resources/example-reports/security-report-template.ini

```diff
@@ -11,16 +11,16 @@ value = Trade.counterparty.value
 value = Trade.settlementDate
 
 [Maturity Date]
-value = Security.product.expiryDate
+value = Security.securityInfo.additionalInfo.ExpiryDate
```

Comment:
This doesn't look great. Is there any scope for shortening names or renaming fields? Something like

    value = Security.info.attributes.ExpiryDate